kotlin/samples/hello-workflow/build.gradle

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+
+android rootProject.ext.defaultAndroidConfig
+
+android {
+  defaultConfig {
+    applicationId "com.squareup.sample.helloworkflow"
+    minSdkVersion 26
```

Comment:
Is there a specific reason for the high min SDK?

Reply:
![Screen Shot 2019-05-21 at 8 55 29 AM](https://user-images.githubusercontent.com/1884445/58111390-3d047280-7ba6-11e9-8593-1fac2dee0c45.png)

Reply:
Just a bit of fun in the sample code.